cmd/system-probe/main.go

```diff
@@ -151,21 +149,6 @@ func gracefulExit() {
 	os.Exit(0)
 }
 
-func handleSignals(exit chan bool) {
```

Comment:
Was this dead code? Where was it being called? 🤔

Reply:
Yeah it was dead :)